ui/src/app/components/object-explorer/object-explorer.component.scss

```diff
@@ -108,9 +108,9 @@
     }
   }
   .right {
-    width: 70%;
+    width: 40%;
```

Comment:
It moves the position of the filter box which I think needed to be moved earlier since I had written "DELETE" instead of "SKIP" (which is shorter). Not needed now.